projects/plugins/jetpack/_inc/client/recommendations/onboarding-utils.ts

```diff
@@ -0,0 +1,47 @@
+import { ONBOARDING_NAME_BY_PRODUCT_SLUG, ONBOARDING_ORDER } from './constants';
+
+/**
+ * Function to get an onboarding for the specific product
+ *
+ * @param {string} productSlug - slug of the product
+ * @returns {string} onboarding name or null if onboarding not found
+ */
+export function getOnboardingNameByProductSlug( productSlug: string ) {
+	const foundIndex = Object.values( ONBOARDING_NAME_BY_PRODUCT_SLUG ).findIndex( slugs =>
+		slugs.includes( productSlug )
+	);
+
+	if ( -1 === foundIndex ) {
+		return null;
+	}
+
+	return Object.keys( ONBOARDING_NAME_BY_PRODUCT_SLUG )[ foundIndex ];
+}
+
+/**
+ * Function to get an onboarding priority
+ *
+ * @param {string} name - onboarding name
+ * @returns {number} the onboarding priority
+ * @throws exception when the requested onboarding is not in the ONBOARDING_ORDER array.
+ */
+function getOnboardingPriority( name: string ) {
+	const index = ONBOARDING_ORDER.indexOf( name );
+
+	if ( -1 === index ) {
+		throw `The onboarding "${ name }" is not included in the ONBOARDING_ORDER`;
```

Comment:
I bought Boost just to see what would happen, and this error got thrown.
![image](https://user-images.githubusercontent.com/65001528/194897590-0b1fd08a-36ef-40f1-8f87-6010b145e630.png)

Seems like even though we don't have an onboarding for Boost, this function still got called and was not handled to where it avoided a fatal error. Perhaps we can return something here like "unknown" or `null` to tell whatever calls it to just send to the summary page instead? Or perhaps not even redirect to the assistant when boost is purchased.

Reply:
CRM has a different site to purchase it on and Social can't be bought yet, but I assume they would react the same way if they were active/purchased through the same workflow

Reply:
Thanks for spotting that one! It's indeed a very crucial one, I'm addressing it.